src/pages/AuthPage.tsx

```diff
@@ -73,8 +74,13 @@ const AuthPage: React.FC = () => {
       return;
     }
 
+    if (!(discordUsername && username && email && password)) {
```

Comment:
Do we want to add first name and last name?

Reply:
Or we can just force first name and last name before signing up for interviews

Reply:
yes first + last here is a good idea